linker/shared/src/test/scala/org/scalajs/linker/testutils/TestIRBuilder.scala

```diff
@@ -134,4 +134,5 @@ object TestIRBuilder {
     MethodIdent(name)
 
   def int(x: Int): IntLiteral = IntLiteral(x)
+  def str(x: String): StringLiteral = StringLiteral(x)
```

Comment:
If you introduce this, "backfill" usage in another commit? (I can find 9 occurrences of `StringLiteral` in linker tests).